assets/questions/reverseWords.js

```diff
@@ -88,6 +96,23 @@ globalData.questions['reverseWords'] = {  // eslint-disable-line dot-notation
           "last reversed word."
         ].join('')
       ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.reverseString',
+      messages: [
+        "Read the instructions closely. Did you overlook something important?",
+        [
+          "It looks like you're reversing the entire string instead of each word. "
+        ].join('')
+      ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.reverseWordOrder',
+      messages: [
+        "Are you sure you're interpreting the instructions correctly?",
```

Comment:
The problem with that is it only works when the student understands the question. The first buggy error occurs because the student is attempting to solve the correct problem but is failing to implement the solution correctly.

Both these errors can be caused by the student is attempting to solve the wrong problem. 
When this is the case, giving the student an example to work out on paper won't work. An example-solution pair works, but is already presented in the instructions.

While I agree that my tone isn't quite right, I'm struggling to find a good way to get the student to go back and look at the instructions more closely without directly stating that's what they should do, and without presenting the student with redundant information.  

I think the ideal solution might actually be "flashing" the instruction box in some way rather than presenting a message, but we don't currently have the infrastructure. Adding callbacks for specific buggy answers might be a good thing overall.